File: week1/day5/algo-palindromes/python/palindrome.py
def palindrome(word):
    word = str(word)

    new_str = ''

    for char in (word):
        if char.isalpha() or char.isnumeric():
            new_str += char.lower()
    

    if new_str == new_str[::-1]:
        return True

    return False

File: week1/day5/algo-palindromes/js/palindrome.js
exports.palindrome = function(word) {

    new_str = String(word).toLowerCase()

    regex = /[^a-z0-9]/gi

    filtered_str = new_str.replace(regex, '')

    // console.log(filtered_str)

    reversed_string = filtered_str.split('').reverse().join('')

    if (filtered_str === reversed_string) {
        return true
    }

    return false

};
